subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/FasterIncrementalAndroidBuildsPerformanceTest.groovy

```diff
@@ -36,7 +36,9 @@ class FasterIncrementalAndroidBuildsPerformanceTest extends AbstractCrossBuildPe
     def "faster incremental build on #testProject (build comparison)"() {
         given:
         runner.testGroup = "incremental android changes"
+        def agpVersionMinor = "4.0"
```

Comment:
How about we call this variable `agpVersion` or `agpTargetVersion`? "Minor" for me suggest somehow that it contains only `5` of `3.5`.

Reply:
I went with `agpTargetVersion`